My_work_2/Фигуры Лиссажу.py

```diff
@@ -0,0 +1,28 @@
+#%%
+
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+#%%
+
+Ax = 2
+Ay = 2
+wx = 2
+wy = 3
+fix = 0
+fiy = 0
+
+t = np.arange(-3.14, 3.15, 0.01)
```

Comment:
How do we set Pi constant properly in numpy? How do we parametrise this code properly to be able to vary curve resolution?

Reply:
And btw how do we run file Фигуры Лиссажу.py in python from the console? ;)

Reply:
There is constant in numpy: np.pi.
I added new variable `step` which controls the density of points.

Reply:
> And btw how do we run file Фигуры Лиссажу.py in python from the console? ;)

There is difficulty for me with this, 'cause I have little knowledge of this and the situation with python on my PC is complex) I guess I'll ask You for help on the tommorow seminar.

Reply:
No problems discussing this issue during the class. The reason I asked this question was just due to file being named in russian and containing the space symbol ' '. General answer would be to 'escape' the space character in the console with eigther ' ' or "\ " - without quotes. And it is kinda good idea to avoid using russian letters while naming any program-related files (scripts, input data, source files, etc ) due to there is a million of ways somesing could go wrong with such source code during executiion. Yeah - this script is just a mere exercise - yet still it's a good idea to use proper code culture guidelines in every project you interact with.